src/main/java/org/zaproxy/zap/extension/jwt/attacks/SignatureAttack.java

```diff
@@ -76,6 +76,9 @@ public class SignatureAttack implements JWTAttack {
 
     private static final String MESSAGE_PREFIX =
             "jwt.scanner.server.vulnerability.signatureAttack.";
+    private static final String INCORRECT_SIGNATURE_KEY = "username";
+    private static final String INCORRECT_SIGNATURE_VALUE = "admin";
```

Comment:
These are not signature keys or values. we can rename them to USERNAME and ADMIN or USERNAME_KEY and USERNAME_VALUE or something appropriate.